README.md

```diff
@@ -253,6 +242,67 @@ clientConfig.getNetworkConfig().getKubernetesConfig().setEnabled(true)
             .setProperty("service-name", "MY-SERVICE-NAME");
 ```
 
+### Outside Kubernetes Cluster
+
+To use **Hazelcast Dummy Client** (`<smart-routing>false</smart-routing>`) you don't need any plugin, it's enough to expose your Hazelcast cluster with a LoadBalancer (or NodePort) service and set its IP/port as the TCP/IP Hazelcast Client configuration. Dummy Client, however, compromises the performance, since all the communication happens against a single Hazelcast member.
+
+To configure **Hazelcast Smart Client**, you need to perform the following steps:
+* Expose each Hazelcast Member POD with a separate LoadBalancer or NodePort service (the simplest way to do it is to install [Metacontroller](https://metacontroller.app/) and [service-per-pod](https://github.com/GoogleCloudPlatform/metacontroller/tree/master/examples/service-per-pod) Decorator Controller)
+* Configure ServiceAccount with ClusterRole having at least `get` and `list` permissions to the following resources: `endpoints`, `pods`, `nodes`, `services`
+* Use credentials from the created ServiceAccount in the Hazelcast Client configuration
```

Comment:
it might be good to give the hints how to get api-token and ca-certificate out of service account secret

Reply:
Added a hint, the more detailed description is in Code Sample (which I'll later link in this README).